pkg/kudoctl/cmd/install/params.go

```diff
@@ -4,13 +4,54 @@ import (
 	"errors"
 	"fmt"
 	"strings"
+
+	"github.com/spf13/afero"
+	"gopkg.in/yaml.v2"
+
+	"github.com/kudobuilder/kudo/pkg/apis/kudo/v1beta1"
+	"github.com/kudobuilder/kudo/pkg/util/convert"
 )
 
-// GetParameterMap takes a slice of parameter strings, parses parameters into a map of keys and values
-func GetParameterMap(raw []string) (map[string]string, error) {
+// GetParameterMap takes a slice of parameter strings and a slice of parameter filenames as well as a filesystem,
+// parses parameters into a map of keys and values
+func GetParameterMap(raw []string, filePaths []string, fs afero.Fs) (map[string]string, error) {
```

Comment:
- changed the signature to put `fs` first
- explained that values are marshalled in godoc
- perhaps `Marshal` would be better than `Wrap`, but I think changing the name is outside the scope of this PR, I'm merely using this pre-existing function
- I extracted the two key pieces from the function so that it's clear on first sight what it's doing
- I don't think it makes sense to pass the values to the original function, since the purpose of the original function is concerned with splitting key and value on the `=` sign, while the function which reads a parameter value file gets the key and value separately from the YAML parser.